app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,7 @@
+class FoodsController < ApplicationController
+  def index
```

Comment:
See my comment in the Facade. How can we get all that we need by making a single call vs calling two separate methods?